### PR TITLE
Adding reference for Erlang and rust comparison

### PR DESCRIPTION
**Pull request by krishnakumar4a4**
source - https://www.infoq.com/articles/rust-erlang-comparison

It is always skeptical for developers to pick a new language and would be trying to understand relatively with what they have already mastered. This articles aims at narrowing the understanding gap between erlang and rustlang by drawing a close analogy that allows both communities try out each others ways and understand them better.

Yes, I am the resource author.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -223,6 +223,7 @@ The main documentation is always the best beginning, so if you haven't read yet,
 | Python          | <ul><li>[Rust for Python Programmers](http://lucumr.pocoo.org/2015/5/27/rust-for-pythonistas/) - Armin Ronacher</li><li>[py2rs](https://github.com/rochacbruno/py2rs) - Bruno Rocha</li></ul> |
 | Ruby            | <ul><li>[Rust for Rubyists](http://www.rustforrubyists.com/) - [Steve Klabnik][]</li></ul> |
 | Swift           | <ul><li>[A Swift guide to Rust](http://faq.sealedabstract.com/rust/) - sealedabstract</li><li>[Rust and Swift](http://www.chriskrycho.com/rust-and-swift.html) - [Chris Krycho][]</li></ul> |
+| Erlang           | <ul><li>[A Comparison between erlang and rust starting from language semantics to runtime features, performance etc..](https://www.infoq.com/articles/rust-erlang-comparison) - [Krishna Kumar Thokala](https://twitter.com/krishnakumart36)</li></ul> |
```
Comment (ctjhoa) [inline README.md]: For authors equity, can you remove your twitter link?

**Comment by ctjhoa** [comment]
Thanks for the contribution!

**Comment by krishnakumar4a4** [comment]
Hi @ctjhoa , I have removed the link now. Please review it.
https://github.com/ctjhoa/rust-learning/pull/131/commits/55bf62c110cb2e908b3dcab12e62c5b0bfa12f3d

File: README.md [hunk 2]
```diff
@@ -223,6 +223,7 @@ The main documentation is always the best beginning, so if you haven't read yet,
 | Python          | <ul><li>[Rust for Python Programmers](http://lucumr.pocoo.org/2015/5/27/rust-for-pythonistas/) - Armin Ronacher</li><li>[py2rs](https://github.com/rochacbruno/py2rs) - Bruno Rocha</li></ul> |
 | Ruby            | <ul><li>[Rust for Rubyists](http://www.rustforrubyists.com/) - [Steve Klabnik][]</li></ul> |
 | Swift           | <ul><li>[A Swift guide to Rust](http://faq.sealedabstract.com/rust/) - sealedabstract</li><li>[Rust and Swift](http://www.chriskrycho.com/rust-and-swift.html) - [Chris Krycho][]</li></ul> |
+| Erlang           | <ul><li>[A Comparison between erlang and rust starting from language semantics to runtime features, performance etc..](https://www.infoq.com/articles/rust-erlang-comparison) - [Krishna Kumar Thokala][]</li></ul> |
```
Comment (ctjhoa) [inline README.md]: Could you remove square brackets on your name too? It's used to reference another section in the README.

**Comment by krishnakumar4a4** [comment]
Hi @ctjhoa ,

I have made the requested changes, Is it good to be merged now?

**Comment by ctjhoa** [comment]
Ok so there is still square brackets remaining. I will merge it and remove them myself. Thanks for your contribution.

**Pull request merged**